responsive-grid-layout/onder/index.html

```diff
@@ -1,42 +1,18 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="global.css" />
-    <link rel="stylesheet" href="responsive.css" />
-    <title>Responsive grid layout exercise</title>
-  </head>
-
-  <body>
-    <div class="container">
-      <div class="header">Header</div>
-      <div class="sidebar sidebar-left">Sidebar left</div>
-      <div class="content">
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-        <div class="content-item"></div>
-      </div>
-      <div class="sidebar sidebar-right">Sidebar right</div>
-      <div class="footer">Footer</div>
-    </div>
-  </body>
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" href="styles.css">
+<title>Grid Layout</title>
+</head>
+<body>
+<div class="container">
+<header>Header</header>
+<sidebar>sidebar-left</sidebar>
```

Comment:
I haven't heard of the `sidebar` element. A `div` would be more appropriate here :)